diff --git a/lib/engines.js b/lib/engines.js
index 38f993d..1dfec8d 100644
--- a/lib/engines.js
+++ b/lib/engines.js
@@ -16,2 +16,2 @@ engines.yaml = {
-  parse: yaml.safeLoad.bind(yaml),
-  stringify: yaml.safeDump.bind(yaml)
+  parse: yaml.load.bind(yaml),
+  stringify: yaml.dump.bind(yaml)
diff --git a/package.json b/package.json
index c5a1fff..21d15a0 100644
--- a/package.json
+++ b/package.json
@@ -37 +37 @@
-    "js-yaml": "^3.13.1",
+    "js-yaml": "^4.2.0",
